css/layout/flexbox.css

```diff
@@ -8,10 +8,15 @@
   flex-direction: column;
 }
 
-.flex-wrap {
+.is-wrap {
   flex-wrap: wrap;
 }
 
+.is-center{
+  justify-content: center;
+  align-items: center;
+}
+
```

Comment:
Similary here class name **is-center** seems like a question, can use better names

Reply:
okay, will change names to be self-explanatory.